modules/bls-dkg/.mocharc.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
For some reasons, the `verifies backup-wallet sig` suite keeps timing out on ci while running in <200ms on local. CI-wise, the tests that are not timed out often run ~1800ms, which is a big difference. Adding a timeout of 5000ms here